build.gradle

```diff
@@ -32,11 +33,16 @@ subprojects {
     apply plugin: 'com.palantir.baseline-eclipse'
     apply plugin: 'com.palantir.baseline-findbugs'
     apply plugin: 'com.palantir.baseline-idea'
+    apply plugin: 'com.palantir.docker-test-runner'
     apply plugin: 'org.inferred.processors'
 
     repositories {
         jcenter()
     }
 
     sourceCompatibility = 1.7
+
+    if (System.env.CIRCLE_TEST_REPORTS) {
+        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
```

Comment:
OK, don't spend too much time on this, it's more of a nice-to-have...

On Mon, Apr 11, 2016 at 5:13 PM Nick Miyake <notifications@github.com>
wrote:

> In build.gradle
> <https://github.com/palantir/http-remoting/pull/92#discussion_r59303170>:
>
> >      apply plugin: 'org.inferred.processors'
> >
> >      repositories {
> >          jcenter()
> >      }
> >
> >      sourceCompatibility = 1.7
> > +
> > +    if (System.env.CIRCLE_TEST_REPORTS) {
> > +        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
>
> I'll double-check. In general, it should be possible to add a
> Circle/Gradle task that just copies the XML artifacts to the expected
> Circle location, in which case I could extract the special Circle logic
> from the Gradle task. I'm sure this would work in the success cases, but
> less sure about whether or not the task would be executed if a test fails.
> Will play around on Circle to test.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/http-remoting/pull/92/files/778de38d91deecec8d1d8164d58699adb3ffdd41#r59303170>
>